kratos/python/add_strategies_to_python.cpp

```diff
@@ -173,17 +175,17 @@ namespace Kratos
             return dummy.CreateEmptyVectorPointer();
         }
 
-        // 	Kratos::shared_ptr< CompressedMatrix > CreateEmptyMatrixPointer()
-        // 	{
-        // 		Kratos::shared_ptr<CompressedMatrix> pNewMat = Kratos::shared_ptr<CompressedMatrix>(new CompressedMatrix() );
-        // 		return pNewMat;
-        // 	}
+        //     Kratos::shared_ptr< CompressedMatrix > CreateEmptyMatrixPointer()
```

Comment:
take out the comments (i mean, erase the commented lines)

Reply:
Ok